jerry-core/api/jerry.c

```diff
@@ -1719,6 +1719,91 @@ jerry_value_to_bigint (const jerry_value_t value) /**< input value */
 #endif /* ENABLED (JERRY_BUILTIN_BIGINT) */
 } /* jerry_value_to_bigint */
 
+/**
+ * Call the toInteger abstrart operation for the given api value.
+ *
+ * @return undefined value - if success
+ *         thrown error - otherwise
+ */
+jerry_value_t
+jerry_value_to_integer (jerry_value_t value, /**< input value */
+                        double *number_p)  /**< [out] double number */
+{
+  jerry_assert_api_available ();
+
+  if (ecma_is_value_error_reference (value))
+  {
+    return jerry_throw (ecma_raise_type_error (ECMA_ERR_MSG (error_value_msg_p)));
+  }
+
+  jerry_value_t return_val = ecma_op_to_integer (value, number_p);
+
+  if (ecma_is_value_error_reference (return_val))
+  {
+    return jerry_return (return_val);
+  }
+
+  return ECMA_VALUE_UNDEFINED;
+} /* jerry_value_to_integer */
+
+/**
+ * Call the ToUint32 abstrart operation for the given api value.
+ *
+ * @return undefined value - if success
+ *         thrown error - otherwise
+ */
+jerry_value_t
+jerry_value_to_uint32 (jerry_value_t value, /**< input value */
+                       uint32_t *number_p) /**< [out] double number */
+{
+  jerry_assert_api_available ();
+
+  if (ecma_is_value_error_reference (value))
+  {
+    return jerry_throw (ecma_raise_type_error (ECMA_ERR_MSG (error_value_msg_p)));
+  }
+
+  ecma_number_t number;
+  value = ecma_op_to_number (value, &number);
+
+  if (ECMA_IS_VALUE_ERROR (value))
+  {
+    return ECMA_VALUE_ERROR;
+  }
+
+  *number_p = ecma_number_to_uint32 (number);
+  return ECMA_VALUE_UNDEFINED;
+} /* jerry_value_to_uint32 */
+
+/**
+ * Call the ToInt32 abstrart operation for the given api value.
+ *
+ * @return undefined value - if success
+ *         thrown error - otherwise
+ */
+jerry_value_t
+jerry_value_to_int32 (jerry_value_t value, /**< input value */
```

Comment:
This should be `const jerry_value_t value` by definition (same in all other functions.)